types/postcss/extensions.d.ts

```diff
@@ -0,0 +1,12 @@
+import * as postcss from 'postcss';
+declare module 'postcss' {
+
+	interface NodeSource {
+		lang?: string;
+
+	}
+
+	interface Input {
+		css?: string;
+	}
```

Comment:
[note] This `css` property is missing the official PostCSS type definitions:

https://github.com/postcss/postcss/blob/ee267d5f6406e9a9ca88be4191ffb6c8b648dc15/lib/input.es6#L25-L34